src/resources/MLModels/MLModelsInterfaces.ts

```diff
@@ -0,0 +1,75 @@
+import {IdAndDisplayNameModel} from '../BaseInterfaces';
+
+export interface MLModel extends MLModelInfo, MLModelCreated {
+    orgId: string;
+    id: string;
+    engineId: string;
+    modelName: string;
+    modelDisplayName?: string;
+    modelCreationTime: number;
+    nextModelUpdateTime?: number;
+    modelVersion?: string;
+    engineVersion?: string;
+    platformVersion?: 1 | 2;
+    versionMatcher?: string;
+    apiKeysThatCanEdit?: IdAndDisplayNameModel[];
+    groupsThatCanEdit?: IdAndDisplayNameModel[];
+    modelErrorDescription?: ModelErrorDescription;
+    previousModelUpdateTime?: number;
+    intervalTime: number;
+    intervalUnit: 'DAY' | 'WEEK' | 'MONTH';
+    exportPeriod: string;
+    exportOffset?: string;
+    status:
+        | 'DELETED'
+        | 'SCHEDULING'
+        | 'SCHEDULED'
+        | 'ERROR'
+        | 'PENDING'
+        | 'REGISTERED'
+        | 'PAUSED'
+        | 'REGISTERING'
+        | 'IN_PROGRESS'
+        | 'ERROR_DELETING'
+        | 'ONLINE'
+        | 'OFFLINE'
+        | 'IN_CREATION';
+    commandLineParameters?: string[];
+    commonFilter?: string;
+    customEventFilter?: string;
+    searchEventFilter?: string;
+    viewEventFilter?: string;
+}
+
+export interface ModelErrorDescription {

```

Comment:
Seems like this is the wrong type
![image](https://user-images.githubusercontent.com/35579930/67868996-240fce00-fb03-11e9-9a6b-7230d0b62750.png)
![image](https://user-images.githubusercontent.com/35579930/67869016-2bcf7280-fb03-11e9-8619-c3c9a5204e70.png)